src/components/variables.css

```diff
@@ -7,3 +7,7 @@
 :root .theme-dark {
   --search-overlays-semitransparent: rgba(42, 46, 56, 0.66);
 }
+
+:root {
+  --editor-footer-height: 27px;
+}
```

Comment:
@magsout I was thinking `src/components/Editor/Editor.css`:

```css
.editor-wrapper  {
  --editor-footer-height: 27px
}
```

What do you think?